yarn.lock

```diff
@@ -3144,6 +3144,20 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@synthetixio/router@npm:^3.3.6":
```

Comment:
We have 2 routers now. Need to dedupe/resolve them. 

![IMG_1607](https://github.com/Synthetixio/synthetix-v3/assets/28145325/9d91ea73-66fc-4a2f-91d1-d51d4da8998b)